projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -57,6 +57,24 @@ public function reregister_menu_items() {
 		ksort( $GLOBALS['menu'] );
 	}
 
+	/**
+	 * Forces Posts menu to WPAdmin for Atomic sites only.
+	 * Overloads `add_posts_menu` in parent class.
+	 *
+	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 */
+	public function add_posts_menu( $wp_admin = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+	}
```

Comment:
Non-blocking: Rather than having an empty function (which may be a bit confusing) what do you think about adding a `return` statement or maybe call to the parent function with a forced `$wp_admin = true` value?

```
public function add_posts_menu( $wp_admin = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
  // Do nothing.
  return;
}
```

or

```
public function add_posts_menu( $wp_admin = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
  // Force WP Admin screens on Atomic.
  parent::add_posts_menu( true );
}
```

Reply:
> adding a return statement

The linter told me off for this 😄 

> call to the parent function with a forced $wp_admin = true value?

I guess we could do this, but we'd be relying on the implementation of the parent not to change which is why I preferred to explicitly do _nothing_ in our function so as to have a more deterministic result. I appreciate that it's unlikely that passing `$wp_admin = true` would result in anything other than WPAdmin links being generated but it's still a small risk.

Overall, I dislike that I'm having to overload the parent implementation. I guess however, the benefits of the small class hierarchy here just about outweigh the downsides.

Reply:
I added the `false` along with some docs. I agree it needed something to make it clearer what we are doing. Thanks for the prompt.